### ISSUE TITLE
OOM in training 训练时发生OOM

### ISSUE THREAD

**Issue by yh0608** [jryoungw/unet_3plus#1]
您好，我在用这个网络训练时发生OOM，请问您的环境和设备是什么，。 
我用的keras2.0.2，显卡是1080Ti 12G显存。
显存占用情况是10773M/11169M.

**Comment by jryoungw** [comment]
Can you write in English? I'm not a chinese speaker.

**Comment by yh0608** [comment]
> Can you write in English? I'm not a chinese speaker.

When I use this network to train , there was a Out Of Memory error occurred. keras version 2.0.2，input=(256,256,3), 1080Ti 12GB Memory.   memory used in nvidia-smi is 10773M/11169M。
Thanks!

`ResourceExhaustedError (see above for traceback): OOM when allocating tensor with shape[10,256,256,160]
	 [[Node: conv2d_35/convolution = Conv2D[T=DT_FLOAT, data_format="NHWC", padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/gpu:0"](activation_4/Relu, conv2d_35/kernel/read)]]
`

**Comment by jryoungw** [comment]
I think you need larger GPU. 1080 Ti 12GB seems small for U-Net+++ to be trained.

**Comment by yh0608** [comment]
> I think you need larger GPU. 1080 Ti 12GB seems small for U-Net+++ to be trained.

Your suggestion is right. I changed batch_size from 10 to 5, it works! Thanks for your reply.
